Index: .pr_agent.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[config]\nmodel = \"groq/llama-3.3-70b-versatile\"\nfallback_models = [\"groq/llama-3.3-70b-versatile\"]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.pr_agent.toml b/.pr_agent.toml
--- a/.pr_agent.toml	(revision f71868c201d2ff1cee42516aa67de0de2a5f2221)
+++ b/.pr_agent.toml	(date 1780396169799)
@@ -1,3 +1,2 @@
 [config]
-model = "groq/llama-3.3-70b-versatile"
-fallback_models = ["groq/llama-3.3-70b-versatile"]
+model = "gpt-4o-mini"
\ No newline at end of file
